docs/monitoring.md

```diff
@@ -175,11 +175,10 @@ daemons.conveyor.throttler.release_waiting_requests.[activity].[rse].[account] (
 
 11) Necromancer
 
-```    
+```bash
```

Comment:
I do not think it makes any sense to mark any of these as `bash` when they have nothing to do with a shell